CloudWorkshop_DIPC_200.md

```diff
@@ -24,23 +24,23 @@ Your will need:
 ### Login into DIPC using Oracle Cloud Services Dashboard
 
 1. In your web browser, navigate to cloud.oracle.com, then click Sign in.
-2. Provide the cloud account: oscnas001 then **\<Enter\>**
-3. Provide your user name and password, then click "Sign In" button. You will land in the Dasboard screen ![](images/200/image200_1.png)
-4. In the "Data Integration Platform Cloud" service box click on the hamburguer menu and then select "Open Service Console" ![](images/200/image200_2.png)
-5. Click on the hamburger menu of the DIPC server assigned to you, then click "Data Integration Platform Console" ![](images/200/image200_3.png)
-You will be navigated to your DIPC server Home page. ![](images/200/image200_4.png)
+2. Provide the cloud account: for example XXXXXXX then **\<Enter\>**.
```

Comment:
Since we are assuming they are doing this in the workshop I would just say for example oscnas001 because saying "for example XXXXXXXXX" does NOT really help.